k6/stress/stress.js

```diff
@@ -0,0 +1,49 @@
+import http from 'k6/http';
+import {check, group, sleep, fail} from 'k6';
+
+export let options = {
+  stages: [
+    {duration: '1m', target: 2},
+    {duration: '2m', target: 50},
+    {duration: '2m', target: 100},
+    {duration: '2m', target: 150},
+    {duration: '2m', target: 200},
+    {duration: '2m', target: 250},
+    {duration: '2m', target: 300},
+    {duration: '2m', target: 400},
+    {duration: '5m', target: 500}
```

Comment:
```suggestion
    {duration: '20s', target: 50},
    {duration: '2m', target: 50},
    {duration: '20s', target: 100},
    {duration: '2m', target: 100},
...
```
- 위와 같은 형태로 작성하셔야 계단형태로 테스트를 할 수 있어요~

Reply:
강의 자료중에 보면 스트레스 테스트는 계단 형태의 그래프를 보여주고 있던데요.
계단형태로 테스트해야 하는 이유가 있을까요?

Reply:
일정 기간 동일한 부하를 주어 반응을 보는거에요. 계속 증가시키기만 하면 어느 시점에 문제가 발생했는지 트래킹하기가 쉽지 않아서요.